composer.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "spryker-eco/avalara-tax-product-option",
+  "type": "library",
+  "description": "AvalaraTaxProductOption module",
+  "license": "MIT",
+  "require": {
+    "avalara/avataxclient": "^21.1.1",
+    "php": ">=7.3",
+    "spryker-eco/avalara-tax-extension": "1.0.0",
+    "spryker/kernel": "^3.30.0",
+    "spryker/money": "^2.0.0",
+    "spryker/util-encoding": "^2.0.0"
```

Comment:
* Add `"spryker/transfer": "^3.25.0"` module to dependencies because of `*getOrFail()` usage.
* Copy the file from the module `Bundles/CategoryStorage/dependency.json`.
* Do the same for the `AvalaraTax` module.

Reply:
Resolved.